### DISCUSSION TITLE
Thumbnail not displayed correctly in embed messages on certain servers

### DISCUSSION THREAD

**Discussion by ElieTaillard** [Rapptz/discord.py#9655] [Q&A] [answered]
When sending an embed message with a thumbnail, the thumbnail is not displayed correctly on certain servers. The message is successfully sent to all servers, but the thumbnail is not visible on most of them.
I've checked that the bot has the necessary permissions and no exception is thrown.

Server 1:
![image](https://github.com/Rapptz/discord.py/assets/54487782/1ae49494-44f9-4d05-bc11-58e1ba1851f4)

Server 2:
![image](https://github.com/Rapptz/discord.py/assets/54487782/3e141b2e-9146-450a-bac1-fda1e0cddb5d)

As you can see, I use `channel.send` in a for loop, using the same parameters for each iteration. Why do I get a different result from one server to another?
https://github.com/ElieTaillard/PSTHC-DiscordBot/blob/1b42b66c5add9694bd91038b3579355e66e922e4/psthc_bot.py#L201-L221

**Comment by LostLuma** [comment, marked as answer]
This happens because you can not re-use a `discord.File` in multiple `Messagable.send` calls, as is stated [in the documentation](https://discordpy.readthedocs.io/en/stable/api.html#discord.File) (read the `Note`!). What you will want to do is construct a new File using the same underlying file (the BytesIO instance, in your case) whenever sending it.

**Reply by ElieTaillard** [reply]
Ok thanks for that clarification. I'll fix my code and let you know if the problem is solved
